src/utils.js

```diff
@@ -10,30 +10,53 @@ export function getNetworkName(chainId) {
 
 export function rpcResult(response) {
   // Some providers don’t wrap the response
-  if (response && response.jsonrpc) {
+  if (response && 'jsonrpc' in response) {
```

Comment:
Interesting; usually you'd want to avoid `in`.

Does a `hasOwnProperty()` not work, because we're traversing a proxy or etc?

Reply:
No I almost used `hasOwnProperty()`, but then I thought I don’t have a reason to invalidate `jsonrpc` if it was on the prototype. Do you see one?

Reply:
Not particularly, but then wouldn't normal property accessing through the `.` work?

Reply:
No, it’s set to `undefined` in some cases :-(